src/nimble.nim

```diff
@@ -663,6 +668,29 @@ proc list(options: TOptions) =
       echoPackageVersions(pkg)
     echo(" ")
 
+proc c_printf(frmt: cstring) {.importc: "printf", header: "<stdio.h>",
+  varargs.}
+
+proc listInstalled(options: TOptions) =
+  let pkgsDir = options.getNimbleDir() / "pkgs"
+  if not existsDir(pkgsDir):
+    raise newException(ENimble, "Nimble dir not installed.")
```

Comment:
Change that to: ``raise newException(ENimble, "Could not find Nimble pkgs dir: " & pkgsDir)``